tsfc/finatinterface.py

```diff
@@ -117,6 +117,9 @@ def convert_finiteelement(element, **kwargs):
 
         return finat.make_quadrature_element(cell, degree, scheme), set()
     lmbda = supported_elements[element.family()]
+    if element.family() == "Real" and element.cell().cellname() in {"quadrilateral", "hexahedron"}:
+        lmbda = None
+        element = ufl.FiniteElement("DQ", element.cell(), 0)
```

Comment:
What does this fix? Does it have any impact on https://github.com/firedrakeproject/firedrake/issues/2095 ?